app/admin/quests/dashboard/[questId]/page.tsx

```diff
@@ -215,7 +216,21 @@ export default function Page({ params }: QuestIdProps) {
             balance_href: task.href,
           },
         };
+      } else if(task.task_type === "customapi"){
+        return {
+          type: "CustomApi",
+          data: {
+            id: task.id,
+            api_name: task.name,
+            api_desc: task.desc,
+            api_href: task.href,
+            api_url: task.url,
```

Comment:
here `task.api_url`.
Now I understand why you have put task.url everywhere, but in our case api_url is the real name of the field in the database, even if you are right, for most fields, there is no prefix before (api_).